src/stylesheets/_context_menu.scss

```diff
@@ -14,13 +14,16 @@
         user-select: none;
         z-index: 99;
 
-        &:hover, &:focus {
+        &:hover,
+        &:focus {
             background-color: $context-menu-color-hover;
             color: $context-menu-color-trigger-hover;
-            transition: all 0.2s ease-out;
```

Comment:
Why is this shorthand removed?

Reply:
Because only `background-color`  and `color` are transitioned, not `all`.

Reply:
If you'd like me to, I can rewrite it as two sets of shorthand for `transition`. The way I've solved it here reuses the duration and timing-method instead of duplicating them. I have no preference though, just let me know what you prefer.